release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2855,6 +2855,73 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+To use this authentication method with your applications, you must complete configuration steps on the cloud provider console and your {product-title} cluster.
+
+//For more information, see xref:../nodes/pods/nodes-pods-short-term-auth.adoc#nodes-pods-short-term-auth-configuring-gcp_nodes-pods-short-term-auth[Configuring {gcp-wid-short} authentication for applications on {gcp-short}].
+
+[id="ocp-4-17-4-enhancements-ansible-operator_{context}"]
+===== ansible-operator upstream version information
+
+* The `ansible-operator` version now shows the corresponding upstream version information. (link:https://issues.redhat.com/browse/OCPBUGS-43836[*OCPBUGS-43836*])
+
+[id="ocp-4-17-4-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, when installing a cluster on an {ibm-title} platform and you wanted to add an existing VPC to the cluster, the {cap-ibm-first} would not adds ports 443, 5000, and 6443 to the security group of the VPC. This situation prevented the VPC from being added to the cluster. With this release, a fix ensures that the {cap-ibm-first} adds the ports to the security group of the VPC so that the VPC gets added to yoru cluster. (link:https://issues.redhat.com/browse/OCPBUGS-44226[*OCPBUGS-44226*])
+
+* Previously, the installation program populated the `network.devices`, `template` and `workspace` fields in the `spec.template.spec.providerSpec.value` section of the {vmw-full} control plane machine set custom resource (CR). These fields should be set in the {vmw-short} failure domain, and the installation program populating them caused unintended behaviors.
++
+Updating these fields did not trigger an update to the control plane machines, and these fields were cleared when the control plane machine set was deleted. With this release, the installation program is updated to no longer populate values that are included in the failure domain configuration. If these values are not defined in a failure domain configuration, for instance on a cluster that is updated to {product-title} {product-version} from an earlier version, the values defined by the installation program are used. (link:https://issues.redhat.com/browse/OCPBUGS-44047[*OCPBUGS-44047*])
+
+* Previously, enabling ESP hardware offload using IPSec on attached interfaces in Open vSwitch broke connectivity due to a bug in Open vSwitch. With this release, OpenShift automatically disables ESP hardware offload on the Open vSwitch attached interfaces, and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-44043[*OCPBUGS-44043*])
+
+* Previously, enabling ESP hardware offload using IPSec on attached interfaces in Open vSwitch broke connectivity due to a bug in Open vSwitch. With this release, OpenShift automatically disables ESP hardware offload on the Open vSwitch attached interfaces, and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-43917[*OCPBUGS-43917*])
```

Comment:
Since these are the same bug with two tickets, can this be combined into one entry, with two bugs listed at the end? just a thought.

Reply:
Good catch. OCPBUGS-44043 targets 4.16 and is not attached to the Images advisory so I'll drop this bug.